### ISSUE TITLE
ABIDE Performance

### ISSUE THREAD

**Issue by tuttyfrutyee** [egyptdj/stagin#8]
Hi great work. I have tried your model on ABIDE dataset and could not get validation accuracy greater than 60% (chance level is around 55%), whereas with SVM I can get to 70% levels. I did optimize and play around with hyperparameters but with no luck. 

To be honest, I would love to hear your speculations on this situation. Do you think the multi-site nature is the source of the problem or just "autism" is not much detectable on fMRI data? Thanks...

**Comment by tuttyfrutyee** [comment]
Great answer, thank you.
